Prague 2019 blog post - events and activities, call for volunteers reminder

thatdocslady opened the pull request.

thatdocslady committed:
Prague 19 blog post - events and activities

thatdocslady committed:
Fix ticket numbers

thatdocslady committed:
Fix yaml and some typos

thatdocslady commented:
@mxsasha feel free to add diversity ticket sponsor info here and I'll work it into the post

mxsasha commented:
For diversity tickets, we had 18 applications and will contact the 10 people who got one soon. The sponsor is https://twitter.com/buildkite :)

mxsasha reviewed (approved):
Looks great! So, we're handing out wristbands for the party at registration this time?

ericholscher commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+:template: {{year}}/generic.html
+
+.. post:: July 17, 2019
```
Need to update this before we ship it.

thatdocslady replied:
```suggestion
.. post:: July 22, 2019
```

ericholscher commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+
+Check out what we have in store for you this year in Prague:
+
+- **Boat Tour of Prague, Saturday September 14th**. If you're in town early, you're invited to join us for a 2-hour cruise on the Vltava river, with light snacks and drinks included. This activity is priced separately and isn't included with the conference ticket, so you're welcome to register your friends and family who might be traveling with you. You can find all the details in our `Boat Tour  ...
```
Think we're calling it a Boat Ride not Tour. Also are we doing snacks/drinks, I don't remember where we landed on this.

thatdocslady replied:
We decided to run a tab and pay per consumption on the spot by credit card :)

ericholscher commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+
+- **Lightning Talks.** One of our favorite traditions, `lightning talks <https://www.writethedocs.org/conf/prague/2019/lightning-talks/>`_ are as entertaining as they are informative. Each block of lightning talks will have 5-6 spots of up to 5 minutes each, both blocks will be after lunch during the main conference days. You can sign up on the mornings of the main conference days and prepare for ...
+
+- **Unconference.** Parallel to the talks, you're welcome to join the `unconference sessions <https://www.writethedocs.org/conf/prague/2019/unconference/>`_. These freestyle sessions are a great opportunity to brainstorm with your fellow documentarians about anything from the latest tooling, the best way to write an email, or how to handle stress and burnout. You can signup for the unconference s ...
```
```suggestion
- **Unconference.** Parallel to the talks, you're welcome to join the `unconference sessions <https://www.writethedocs.org/conf/prague/2019/unconference/>`_. These freestyle sessions are a great opportunity to brainstorm with your fellow documentarians about anything from the latest tooling, the best way to write an email, or how to handle stress and burnout. You can signup for the unconference sessions during the Writing Day and throughout the conference.
```

ericholscher commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+
+- **Unconference.** Parallel to the talks, you're welcome to join the `unconference sessions <https://www.writethedocs.org/conf/prague/2019/unconference/>`_. These freestyle sessions are a great opportunity to brainstorm with your fellow documentarians about anything from the latest tooling, the best way to write an email, or how to handle stress and burnout. You can signup for the unconference s ...
+
+- **Social Event.** This year our social event will be held at `RKO (Restaurant by Retro), Francouzská 4, Prague 2 <https://goo.gl/maps/qJUvEuShp8kUC7ac8>`_, a bar with a great vibe owned by Retro Music Hall. The bar is about 15-20 minutes walk from the main conference venue, and is extremely well-connected by metro and tram service (Namesti Miru stop). The event will start at 20:00, and light sn ...
```
Not a huge fan of wristbands. I think they are just a proxy for badges, and badges are better since they have your name on it. Is there a reason we're changing this?

thatdocslady replied:
We did this as an experiment last year to accommodate for +1s and also so that people don't have to wear their badge during the party, since their name is fully visible on the badge.

thatdocslady replied:
The point is to give people a choice if they want to wear their badge or wristband, some people are concerned about privacy displaying their name outside the conference venue, and also not have to worry about +1s not having a badge.

thatdocslady replied:
Removed the specific text for now, we can work it out later ;)

thatdocslady commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+
+Check out what we have in store for you this year in Prague:
+
+- **Boat Tour of Prague, Saturday September 14th**. If you're in town early, you're invited to join us for a 2-hour cruise on the Vltava river, with light snacks and drinks included. This activity is priced separately and isn't included with the conference ticket, so you're welcome to register your friends and family who might be traveling with you. You can find all the details in our `Boat Tour  ...
```
```suggestion
- **Boat Ride, Saturday September 14th**. If you're in town early, you're invited to join us for a 2-hour cruise on the Vltava river, with light snacks and drinks included. This activity is priced separately and isn't included with the conference ticket, so you're welcome to register your friends and family who might be traveling with you. You can find all the details in our `Boat Ride <https://www.writethedocs.org/conf/prague/2019/outing/>`_ page, or register directly on our `ticket website <https://ti.to/writethedocs/write-the-docs-prague-2019>`_.
```

thatdocslady committed:
Apply suggestions from code review

Feedback

Co-Authored-By: Eric Holscher <25510+ericholscher@users.noreply.github.com>

thatdocslady commented:
@mxsasha I just checked the previous blog posts and we already mentioned Buildkite as a sponsor. Also, since the diversity ticket emails have already been sent, seems a bit late to mention this in the post. Any strong objections for leaving this out of this one?

thatdocslady committed:
Merge branch 'master' into prague19-blog-events

mxsasha commented:
> @mxsasha I just checked the previous blog posts and we already mentioned Buildkite as a sponsor. Also, since the diversity ticket emails have already been sent, seems a bit late to mention this in the post. Any strong objections for leaving this out of this one?

I think we can leave it out.

thatdocslady commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+
+- **Unconference.** Parallel to the talks, you're welcome to join the `unconference sessions <https://www.writethedocs.org/conf/prague/2019/unconference/>`_. These freestyle sessions are a great opportunity to brainstorm with your fellow documentarians about anything from the latest tooling, the best way to write an email, or how to handle stress and burnout. You can signup for the unconference s ...
+
+- **Social Event.** This year our social event will be held at `RKO (Restaurant by Retro), Francouzská 4, Prague 2 <https://goo.gl/maps/qJUvEuShp8kUC7ac8>`_, a bar with a great vibe owned by Retro Music Hall. The bar is about 15-20 minutes walk from the main conference venue, and is extremely well-connected by metro and tram service (Namesti Miru stop). The event will start at 20:00, and light sn ...
```
```suggestion
- **Social Event.** This year our social event will be held at `RKO (Restaurant by Retro), Francouzská 4, Prague 2 <https://goo.gl/maps/qJUvEuShp8kUC7ac8>`_, a bar with a great vibe owned by Retro Music Hall. The bar is about 15-20 minutes walk from the main conference venue, and is extremely well-connected by metro and tram service (Namesti Miru stop). The event will start at 20:00, and light snacks and drinks will be provided while our tab lasts. If you want a sit-down dinner we recommend to grab something on the way before we kick off the evening. **Entrance is only for conference attendees**.
```

thatdocslady committed:
Update docs/conf/prague/2019/news/events-activities.rst

Removing wristband/badge reference for now, we can pick up the discussion later but I don't want to block the post because of this

thatdocslady commented on docs/conf/prague/2019/news/events-activities.rst:
```diff
@@ -0,0 +1,62 @@
+
+- **Social Event.** This year our social event will be held at `RKO (Restaurant by Retro), Francouzská 4, Prague 2 <https://goo.gl/maps/qJUvEuShp8kUC7ac8>`_, a bar with a great vibe owned by Retro Music Hall. The bar is about 15-20 minutes walk from the main conference venue, and is extremely well-connected by metro and tram service (Namesti Miru stop). The event will start at 20:00, and light sn ...
+
+- **Job Fair.** Following the success last year, we're happy to announce that during Tuesday morning, part of the unconference space will be converted to the job fair. The goal is to give job-seeking documentarians an opportunity to connect with potential employers. Documentarians looking for jobs will be able to visit employers at their table to learn about the company culture, ask about specifi ...
```
```suggestion
- **Job Fair.** Following the success last year, we're happy to announce that during Tuesday morning, part of the unconference space will be converted to the job fair. The goal is to give job-seeking documentarians an opportunity to connect with potential employers. Documentarians looking for jobs will be able to visit employers at their table to learn about the company culture, ask about specific job openings and talk to current employees. If you're an employer looking to fill a role, you can find information about reserving a table in the `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_.
```

thatdocslady committed:
Update docs/conf/prague/2019/news/events-activities.rst
